app/src/main/java/org/oppia/app/splash/SplashFragment.kt

```diff
@@ -0,0 +1,30 @@
+package org.oppia.app.splash
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import org.oppia.app.HomeActivity
+import org.oppia.app.R
+
+/** This SplashFragments navigates to Home page once the app is finished loading completely*/
+
+class SplashFragment : Fragment() {
+
+  override fun onCreateView(
+    inflater: LayoutInflater, container: ViewGroup?,
+    savedInstanceState: Bundle?
+  ): View? {
+    val view = inflater.inflate(R.layout.splash_fragment, container, false)
```

Comment:
Do we actually need a view for this fragment? If we're just using the content view of the activity for the transition, let's remove the extra splash fragment layout since it's not being used for anything.

Reply:
Also, are we still setting the splash page drawable anywhere? I'm not noticing it.

Reply:
@veena14cs can you address this comment?

Reply:
Since it was design from Technical Document's perspective I had added fragment but as its no where used I am removing it

Reply:
> Also, are we still setting the splash page drawable anywhere? I'm not noticing it.

splash page was earlier used through custom drawable file `bg_splash.xml` now I am setting it directly to style.